source/1_engine/rex_engine/src/diagnostics/logging/log_category.cpp

```diff
@@ -19,17 +19,17 @@ namespace rex
   //-------------------------------------------------------------------------
   bool LogCategory::is_async() const
   {
-      return m_is_async == IsAsync::yes;
+    return m_is_async == IsAsync::yes; // NOLINT (readability-implicit-bool-conversion)
```

Comment:
it's actually best NOT to store these arguments as yesno enums, yesno enums should only be used as params.
this is to avoid the branching internally when it needs to get converted to a bool.
that'll also solve the clang-tidy warning emitted here